pkg/buganalysis/cache.go

```diff
@@ -21,7 +21,7 @@ import (
 // It is stateful though, so for a time after clearing the data will not be up to date until the Update is called
 type BugCache interface {
 	ListJobBlockingBugs(job string) []bugsv1.Bug
-	ListBugs(release, variant, testName string) []bugsv1.Bug
+	ListBugs(release, variant, testName string, invertReleaseQuery bool) []bugsv1.Bug
```

Comment:
Prefer a second method that can call a private helper with this bool.  That keeps code paths consistent and makes intent in the calling code very clear and searchable.

Reply:
updated